### PR TITLE
Switch Heroku page to "request access" and other fixes

### PR DESCRIPTION
**Pull request by brikis98**
1. The CTA is now "request access" and the pricing and contact pages, as well as the FAQ, mention this is a private beta.
1. Expose "what plan are you interested in" radio buttons on the contact page so the user can see / change their selection.
1. Change footer to a checklist. The HIPAA landing page had this and I liked the way it looked.
1. Try to fix the thank you page. I _think_ the `permalink` must end with a trailing slash to work in prod. Not sure of the reason.

Preview env: https://deploy-preview-466--keen-clarke-470db9.netlify.app/migrate-from-heroku-to-aws/

### CODE REVIEW

**Comment by ina-stoyanova** [comment]
The PR looks good, but I just gave the thank you page a go, and it still seems to break. I'm also seeing `ERROR for site owner:
Invalid domain for site key` on the contact-us form page come up (I guess it pops up from netlify).

**Comment by ina-stoyanova** [comment]
Nice touch with footer & the rest of the tweaks! Really like how it looks now!

**Review by ina-stoyanova — APPROVED** [review]
Approving to unblock you, with the caution that the thank you page link still isn't working.

**Comment by brikis98** [comment]
Thanks for the review! Will merge now and revisit the thank you page. BTW, what testing did you do that showed it isn't working?

**Comment by ina-stoyanova** [comment]
I tried to follow the flow of the pages - click on "request early access", fill in my details and hope to see the thank you page.  Should I be doing something else instead?

**Comment by brikis98** [comment]
That seems right, but it worked fine for me... And I just tested in prod with the newly deployed code and that worked fine too. Only issue I found was a broken link in the footer. Fix coming shortly.

**Comment by ina-stoyanova** [comment]
Oh..... I wonder why it didn't work for me - I clicked on the link in the PR description, and tried it two or three times. Let me try again.

**Comment by ina-stoyanova** [comment]
Huh. Whatever I was seeing, it's not no happening, and all works as a charm.

**Comment by brikis98** [comment]
Weird!
